internal/account/service.go

```diff
@@ -47,11 +54,37 @@ func (a *AccountService) RegisterAccount(ctx context.Context, spec RegisterAccou
 	return a.accountDBAccessor.RegisterAccount(ctx, account)
 }
 
+func (a *AccountService) Login(ctx context.Context, spec AccountCredentialSpec) (string, error) {
+	var loginError = fmt.Errorf("login failed")
```

Comment:
just put this on global and wrap the error
```suggestion
	var ErrLoginFailed = errors.New("login failed")
```